voc/requirement.go

```diff
@@ -25,6 +27,8 @@
 
 package voc
 
-type OSLog struct {
-	*Log
+type Requirement struct {
```

Comment:
can also be deleted

Reply:
I deleted the Requirement in the Ontology.